tools/test/examples/examples.py

```diff
@@ -27,17 +28,20 @@
 """ import and bulid a bunch of example programs """
 
 ROOT = abspath(dirname(dirname(dirname(dirname(__file__)))))
-sys.path.insert(0, ROOT)
+DEFAULT_BUILD_PROFILES = [
+    "develop",
+    "mbed-os/tools/profiles/extensions/minimal-printf.json",
```

Comment:
+1 for introducing profiles (might get release as separate PR 👍 )

Reply:
That is a good one as we might use this in near future (I wonder if this can be enabled in nightly right away, 
cc @OPpuolitaival )

Reply:
@0xc0170 we agreed that this PR will be merged after 5.14 because we don't want to introduce the change now. That's great if it can be enabled in nightly right away. 
We also need to build PDMC with `minimal-printf` profile extension. There is an email thread regarding this

Reply:
I set to do not merge. Once 5.14.0 is out, we will remove the label and integrate

Reply:
@0xc0170,
The `do not merge` label can now be removed.